docs/user-interface/controls/twopaneview.md

```diff
@@ -0,0 +1,120 @@
+monikerRange: ">= net-maui-7.0"
+author: conceptdev
+ms.author: crdun
+ms.date: 11/03/2022
+---
+# .NET MAUI TwoPaneView layout
+
+[![Browse sample.](~/media/code-sample.png) Browse the sample](/samples/dotnet/maui-samples/userinterface-controls-twopaneview/)
+
+The `TwoPaneView` class represents a container with two views that size and position content in the available space, either side-by-side or top-to-bottom. `TwoPaneView` inherits from `Grid` so the easiest way to think about these properties is as if they are being applied to a grid.
+
+:::image type="content" source="media/twopaneview/foldable-maui-app.png" alt-text="Surface Duo dual-screen emulator showing a basic TwoPaneView test app":::
+
+The layout control is provided by the [Microsoft.Maui.Controls.Foldable NuGet package](https://www.nuget.org/packages/Microsoft.Maui.Controls.Foldable/).
+
+## Foldable device support overview
+
+Foldable devices include the Microsoft Surface Duo and Android devices from other manufacturers. They bridge the gap between phones and larger screens like tablets and desktops because apps might need to adjust to a variety of screen sizes and orientations on the same device, including adapting to a hinge or fold in the screen.
+
+Visit the [dual-screen developer docs](/dual-screen/) for more information about building apps that target foldable devices, including [design patterns and user experiences](/dual-screen/design/). There is also a [Surface Duo emulator](/dual-screen/android/emulator/) you can download for Windows, Mac, and Linux.
+
+> [!IMPORTANT]
+> The `TwoPaneView` control only adapts to Android foldable devices that support the Jetpack Window Manager API provided by Google (such as Microsoft Surface Duo).
+>
+> On all other platforms and devices (i.e. other Android devices, iOS, macOS, Windows) it acts like a configurable and responsive split view that can dynamically show one or two panes, proportionally sized on the screen.
+
+## Add and configure the Foldable support NuGet
+
+1. Open the **NuGet Package Manager** dialog for your solution.
+2. Under the **Browse** tab, search for `Microsoft.Maui.Controls.Foldable`.
+3. Install the `Microsoft.Maui.Controls.Foldable` package to your solution.
+4. Add the `UseFoldable()` initialization method (and namespace) call to the project's `MauiApp` class, in the `CreateMauiApp` method:
+
+    ```csharp
+    using Microsoft.Maui.Foldable; // ADD THIS NAMESPACE
+    ...
+    public static MauiApp CreateMauiApp()
+    {
+        var builder = MauiApp.CreateBuilder();
+        ...
+        builder.UseFoldable(); // ADD THIS LINE TO THE TEMPLATE
+        return builder.Build();
+    }
+    ```
+
+    The `UseFoldable()` initialization is required for the app to be able to detect changes in the app's state, such as being spanned across a fold.
+
+5. Update the `[Activity(...)]` attribute on the `MainActivity` class in **Platforms/Android**, so that it includes _all_ these `ConfigurationChanges` options:
```

Comment:
```suggestion
5. Update the `[Activity(...)]` attribute on the `MainActivity` class in *Platforms/Android*, so that it includes _all_ these `ConfigurationChanges` options:
```

Reply:
Italics for files/folder names.

Reply:
😭